nwcsafpps_runner/tests/test_config.py

```diff
@@ -126,9 +125,9 @@ def test_read_pps_runner_config(self, gethostname, config):
         """Test loading and initialising the yaml config"""
         gethostname.return_value = "my.local.host"
         config.return_value = self.config_pps_complete
-        myconfig_filename = '/tmp/my/config/file'
+        myconfig_filename = '/tmp/my/config/file.yaml'
```

Comment:
Should we introduce pathlib here perhaps, getting rid of the codefactor warning?

Reply:
maybe tmp_path can be used here!